databaseutil/databaseutil.go

```diff
@@ -3,19 +3,73 @@ package databaseutil
 import (
 	"database/sql"
 	// Notice that we’re loading the driver anonymously, The driver registers itself as being available to the database/sql package.
+	"fmt"
 	_ "github.com/lib/pq"
+	"golang.org/x/crypto/bcrypt"
+	"time"
 )
 
-func Connect(dbUrl string) (*sql.DB, error) {
-	db, err := sql.Open("postgres", dbUrl)
+var db *sql.DB
+
+func Connect(dbUrl string) error {
+	temp, err := sql.Open("postgres", dbUrl)
 	if err != nil {
-		return nil, err
+		return err
 	}
 
+	db = temp
+
 	// Quickly test if the connection to the database worked.
 	if err := db.Ping(); err != nil {
-		return nil, err
+		return err
+	}
+
+	return nil
+}
+
+func SaveNewUser(
+	displayName string,
+	emailAddress string,
+	password string) (int64, error) {
+
+	hash, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
```

Comment:
hashedPassword it is. Because i don't consider this hash to be an inherent proptery of the password (since it can be hashed with many different hashing functions). And I feel `passWordHash` implies its proptery-esc, while hashedPassword seems like a function that has been applied password.